autokeras/blocks/basic.py

```diff
@@ -375,6 +551,93 @@ def build(self, hp, inputs=None):
         return model.outputs
 
 
+class TokenAndPositionEmbedding(block_module.Block):
```

Comment:
I still feel we should make it part of the Transformer block, unless it is useful by any other well know models.